### PR TITLE
Mergeable with caveats: Data-blog article #1. See #636 for checklist.

### PR DESCRIPTION
**Pull request by 4kbt**
Creating PR at @awong-dev's suggestion in order to see how things roll in Heroku.

### CODE REVIEW

File: views/datablog.handlebars [hunk 1]
```diff
@@ -0,0 +1,330 @@
+<div class="translated-content">
+  {{> top_nav }}
+
+  <div class="container mt-5">
+    <div class="row">
+      <div class="col">
+	<head><title>Insights from FindTheMasks-US Data</title>
```
Comment (nburt) [inline views/datablog.handlebars:7]: happy to update this if needed but you'll need to set this in the layout. It'll need to be passed through as a local (like "ogTitle") and then the static.handlebars file will need to use it or fallback to a default title.

**Comment by nburt** [comment]
This looks awesome! Given that this is a US focused analysis I'm guessing we don't need to translate it right?

**Comment by 4kbt** [comment]
Correct; I haven't begun to think about translating it.

File: views/datablog.handlebars (hunk 1 above)
Comment (4kbt) [reply views/datablog.handlebars:7]: You're going to be able to do this sort of thing much more efficiently than I can. If we make a second article, I'll be able to be a copy-paste monkey the next go-round.

**Comment by 4kbt** [comment]
PR is now mergeable with caveats. See #636 for the remaining tasks.

**Pull request merged**